apps/api/src/libs/abstract/abstract.repository.ts

```
@@ -84,6 +85,26 @@ export abstract class AbstractRepository<TDocument extends AbstractDocument> {
+        filterQuery,
+        updateQuery,
+        queryOptions,
+        session,
+    }: {
+        filterQuery: FilterQuery<TDocument>;
+        updateQuery: UpdateQuery<TDocument>;
+        queryOptions?: Partial<MongooseUpdateQueryOptions<TDocument>>;
+
+        session?: ClientSession;
+    }): Promise<boolean> {
+        delete updateQuery?._id;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.



(lint/performance/noDelete)